easybuild/easyconfigs/s/Singularity/Singularity-2.3.1-GCC-5.4.0-2.26.eb

```diff
@@ -22,7 +22,7 @@ sanity_check_paths = {
     'dirs': ['etc', 'libexec/singularity'],
 }
 
-# next steps after instalations
+# next steps after installation
 # INSTALATION_PATH=your_instalation_path
```

Comment:
No problem. I should have caught those before. At least the previous author was consistent!